ocs_ci/ocs/dr/dr_workload.py

```diff
@@ -1046,6 +1051,221 @@ def validate_data_integrity_vm(
         ), f"Failed: MD5 comparison after {app_state}"
 
 
+class Busybox_DiscoveredApps(DRWorkload):
```

Comment:
Can we have CamelCase for class names ?

```suggestion
class BusyboxDiscoveredApps(DRWorkload):
```

Reply:
Done